ansible/ci-deployment.yml

```diff
@@ -33,8 +47,26 @@
     - role: omero-web-runtime
     - role: nginx
 
+
 # Deploy Documentation prerequisites
 - hosts: ci-docs
   roles:
+    - role: virtualenv
     - role: sphinx-build
     - role: ansible
+
+  vars:
+    virtualenv_user: "{{ jenkinsuser }}"
+    virtualenv_path: /home/{{ jenkinsuser }}/virtualenv
+    virtualenv_site_packages: "yes"
+    virtualenv_pip_rpm_dependencies:
+    virtualenv_pip_dependencies:
```

Comment:
Briefly discussed with @aleksandra-tarkowska, I think the issue is that different jobs have different strategies. Some of them use Jenkins Ant + globally installed Sphinx (BIOFORMATS build jobs) others use globally installed Ant + virtualenv  (OMERO docs autogen jobs). Rationalizing might be a long term goal. To maintain compatibility, my feeling is that installing Sphinx in the system packages via `sphinx-build` and creating a virtualenv with `--system-site-packages` should maintain compatibility for both types of documentation jobs.